include/openspace/navigation/pathcurve.h

```diff
@@ -34,21 +35,35 @@ class Waypoint;
 
 class PathCurve {
 public:
+    struct InsufficientPrecisionError : public ghoul::RuntimeError {
+        explicit InsufficientPrecisionError(std::string msg);
+    };
+
+    struct TooShortPathError : public ghoul::RuntimeError {
+        explicit TooShortPathError(std::string msg);
+    };
+
     virtual ~PathCurve() = 0;
 
     double length() const;
```

Comment:
Mentioning here the unit of the length?  I'm assuming it'd be "meters" but just in case